### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Allanpt**
- [DEMO LINK](https://Allanpt.github.io/layout_stop-watch/)

- [x] Keyframes implemented using from/to + transform with rotate property
- [x] Stopwatch is centered and has correct arrows size
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1 +1,72 @@
...
+  background-color: blue;
+  left: 25%;
+
+  animation: hour-pointer 60s infinite linear;
+  transform-origin: right center;
+}
+
+@keyframes minute-pointer{
+  from {
+    transform: rotate(90deg) translate(-0.1%, -0.1%);
```
Comment (jv-aquino) [inline src/styles/main.scss:54]: The translate here is probably unnecessary

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="yellow-dot"></div>
+      <div class="minute-pointer"></div>
+      <div class="hour-pointer"></div>
```
Comment (jv-aquino) [inline src/index.html:15]: You could use BEM naming conventions here

**Review by jv-aquino — CHANGES_REQUESTED** [review]
Nice and concise code, following the guidelines of a good front end dev, there's just two things that you should check out

**Comment by Allanpt** [comment]
Ok, i think it's done now LOLOLOLOLO

**Review by pedro-ruas — APPROVED** [review]
Great Job!

**Pull request closed without merging**